applications.json

```diff
@@ -7224,6 +7224,18 @@
       "languages" : [
         "swift"
       ]
+    },
+    {
+      "repo_url" : "https://github.com/macmade/Notarize",
+      "title" : "Notarize",
+      "screenshots" : [
+        "https://github.com/macmade/Notarize/raw/master/Assets/MainWindow.png"
```

Comment:
Please change ling from https://github.com/macmade/Notarize/raw/master/Assets/MainWindow.png to https://raw.githubusercontent.com/macmade/Notarize/master/Assets/MainWindow.png due to  CI warning.

Reply:
Ok, done 👍